pkg/helmraiser/helm.go

```diff
@@ -0,0 +1,147 @@
+package helmraiser
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	jsonnet "github.com/google/go-jsonnet"
+	"github.com/google/go-jsonnet/ast"
+	"github.com/pkg/errors"
+	yaml "gopkg.in/yaml.v3"
+)
+
+var deferTempFiles []string
```

Comment:
Package level globals make it impossible to use the same package twice at the same time in a code-base.

See my other comments how to not need this